sormas-base/pom.xml

```diff
@@ -584,10 +584,22 @@
 				<version>${slf4j.version}</version>
 				<scope>provided</scope>
 			</dependency>
+			<dependency>
+				<groupId>ca.uhn.hapi.fhir</groupId>
+				<artifactId>hapi-fhir-structures-r4</artifactId>
+				<version>5.7.0</version>
+				<scope>runtime</scope>
+			</dependency>
 
 			<!-- *** Domain libs END *** -->
 
 			<!-- *** Compile dependencies *** -->
+			<dependency>
```

Comment:
shouldn't this also be provided as a serverlib in the domain?

Reply:
https://github.com/hzi-braunschweig/SORMAS-Project/issues/8324